documentation/staging/content/userguide/managing-domains/_index.md

```diff
@@ -166,7 +166,33 @@ The operator can automatically override WebLogic Server, domain, and introspecto
 This occurs if the Domain `logHomeEnabled` field is explicitly set to `true`, or if `logHomeEnabled` isn't set
 and `domainHomeSourceType` is set to `PersistentVolume`.  When overriding, the log location will be the location specified by the `logHome` setting.
 
-By default, all the log files are located follow the server layout subdirectories when of `logHome` and `logHomeEnabled` is setup.  If you want all the files to locate at a single level at the root of the `logHome`, you can specify `logHomeLayout` as `flat`.
+Server log files are placed in a subdirectory `servers/<server name>/logs`
+by default when both `logHome` and `logHomeEnabled` are set.
+Set `logHomeLayout` to `FLAT` to place all log files at the `logHome` root.
+
+For example, the default layout of the log files under the `logHome` root.
```

Comment:
For example, here is the default layout of the log files under the `logHome` root: